Add SASS Version

Adrian-Mazaud opened the pull request:
@mobyou 
Salut c'est du pure SASS, dis moi si tu veux une version scss, tout fonctionne en faisant appel à des mixins, d'ailleurs comme tu m'as dis que tu étais sur une V2 j'ai viré pas mal de trucs, flexbox, box-shadow, enfin je n'ai pas trop bien compris le rôle des bicols et biblocs triblocs, ils sont là pour éviter de créer des grilles à chaque fois j'imagine ? Du coup je les rangés dans un dossier legacy, à part...

N'hésites pas à me dire ce qui ne convient pas,
a+

Adrian-Mazaud committed:
Add SASS Version

mobyou reviewed (commented):
Adrian SASS version of do-css

Adrian-Mazaud committed:
fixes @import in @if statement

Petit problème d'@import dans les conditions @if ce qui est interdit, qui concernait $reset, et $legacy.

mobyou reviewed (approved):
Sass version
